src/main/scala/zio/prelude/GenFs.scala

```diff
@@ -24,15 +24,6 @@ object GenFs {
         Gen.causes(gen, Gen.throwable)
     }
 
-  /**
-   * A generator of `Option` values.
-   */
-  val option: GenF[Random with Sized, Option] =
-    new GenF[Random with Sized, Option] {
-      def apply[R1 <: Random with Sized, A](gen: Gen[R1, A]): Gen[R1, Option[A]] =
-        Gen.option(gen)
-    }
-
```

Comment:
I added this on my previous PR, but just realized there's already a `GenF` for `Option` in zio-test

Reply:
Yes, I think ideally all of the instances other than the ones for ZIO Prelude data types would like in ZIO Test, but sometimes we need them here temporarily until we can get a ZIO release that has them.